examples/basic-nextjs/tsconfig.json

```diff
@@ -2,14 +2,28 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "components/*": ["src/components/*"],
-      "lib/*": ["src/lib/*"],
-      "temp/*": ["src/temp/*"],
-      "assets/*": ["src/assets/*"],
+      "components/*": [
+        "src/components/*"
+      ],
+      "lib/*": [
+        "src/lib/*"
+      ],
+      "temp/*": [
+        "src/temp/*"
+      ],
+      "assets/*": [
+        "src/assets/*"
+      ]
```

Comment:
Let's add .sitecore alias for basic-nextjs, basic-pages-router, basic-skatepark
https://github.com/Sitecore/content-sdk/blob/dev/packages/create-content-sdk-app/src/templates/nextjs-app-router%20(beta)/tsconfig.json#L17-L19